servicetest/conftest.py

```diff
@@ -115,3 +115,17 @@ def assert_no_proxy():
     assert grep_for_dbus_proxy() != 0, "dbus-proxy is alive when it shouldn't be"
     yield
     assert grep_for_dbus_proxy() != 0, "dbus-proxy is alive when it shouldn't be"
+
+
+@pytest.fixture(scope="module")
+def create_testoutput_dir(request):
+    """ Create a directory for the generated test files.
+
+        This directory is ignored by git but it's nice to have
+        somewhere locally to store test output to support
+        troubleshooting etc.
```

Comment:
This comment is now a bit off. There is nothing that automatically makes git ignore the directory created by this fixture. Previously it was a comment about the local situation in each test suite directory. Perhaps this comment instead should say that it's recommended by tests to put all their ephemeral test files/output in this dir and have git ignore it. Since that's the intention but it's not necessarily  the case.